testing/load-aware/test.py

```diff
@@ -65,7 +65,7 @@ def prepare_ci():
     install_ocp_pipelines()
 
     run.run("./run_toolbox.py from_config cluster capture_environment --suffix sample")
-
+    run.run("./run_toolbox.py load_aware deploy_trimaran")
```

Comment:
another possible update: add a configuration flag to `testing/pipelines/config.yaml` with: `deploy_trimaran: false`, and if set to true (either from the PR config or from the `load-aware` preset), call this command

Reply:
Also will be included in improvements PR!